noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -10,7 +10,7 @@ use crate::{
 };
 
 /// Marks a contract as an Aztec contract, generating the interfaces for its functions and notes, as well as injecting
-/// the `sync_notes` utility function PXE requires in order to discover notes.
+/// the `sync_private_state` utility function PXE requires in order to discover notes.
```

Comment:
This comment is outdated - PXE no longer requires this fn to discover notes (indeed PXE does not do any discovery of its own). All `sync_private_state` is, well, sync the contract's private state with the chain (including finding new notes, events, etc.). It is typically not needed because contract fns do this automatically, but you may want it if e.g. querying for raw events or notes from pxe's storage.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/14442